src/main/java/com/devcycle/sdk/server/local/api/DevCycleLocalClient.java

```diff
@@ -212,7 +213,7 @@ public <T> Variable<T> variable(DevCycleUser user, String key, T defaultValue) {
             if (beforeError != null) {
                 throw beforeError;
             }
-            evalHooksRunner.executeAfter(reversedHooks, hookContext, variable);
+            evalHooksRunner.executeAfter(reversedHooks, hookContext, sdkVariable);
```

Comment:
could we add the evals and everything to the sdkVariable and pass that through to the hooks? actually, is there a reason for not adding that in through to the hooks?

Reply:
updated the approach to have another param of `VariableMetadat` to maintain compatibility between Cloud & Local bucketing && ensure the hooks always have access to the `Variable`, even in the scenario where the Variable has defaulted